src/events.js

```diff
@@ -158,7 +186,10 @@ const handlers = {
       help: sendHelp,
       thx: sayThankyou,
       thanks: sayThankyou,
-      thankyou: sayThankyou
+      thankyou: sayThankyou,
+      reset: handleReset,
+      '++': doNothing,
+      '--': doNothing,
```

Comment:
giving the bot points also results in the bot trying to run the `++` as an app command, so we get a slack message about the bot not being very smart. The `doNothing` handler fixes that, in theory